lib/ClimaLandSimulations/src/fluxnet_simulation.jl

```diff
@@ -1,4 +1,33 @@
-export fluxnet_simulation
+export fluxnet_simulation, fluxnet_simulation_test

```

Comment:
I think it would be nice if the files in `src/` contained modules, so that namespaces are more clearly defined/isolated. This is similar to what we do currently in ClimaLSM, where we have different modules in `src/` for the different shared utilities, Bucket, Soil, etc.

We could have a `Fluxnet` module for `fluxnet_simulation.jl` and `met_drivers_FLUXNET.jl`, `InputOutput` for `climaland_output_dataframe.jl` and `inputs_dataframe.jl`, and then maybe `Setup` for `domain_setup.jl` and `timestepper_setup.jl` (we should make sure the domain_setup module doesn't have the same name as ClimaLand.Domains). What do you think?

Reply:
Agreed!